app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
-    <string name="app_name">FrettirTEST2</string>
+    <string name="app_name">Frétta Appið</string>
```

Comment:
changing the name of the app to something more nice :)

please change it again, if you have some better idea